src/stream/src/from_proto/source/trad_source.rs

```diff
@@ -85,31 +85,6 @@ impl ExecutorBuilder for SourceExecutorBuilder {
                     }
                 }
 
-                {
-                    // compatible code: handle legacy column `_rw_kafka_timestamp`
-                    // the column is auto added for all kafka source to empower batch query on source
-                    // solution: rewrite the column `additional_column_type` to Timestamp
```

Comment:
I put the compatibility code here to prevent changing the catalog. I don't see how to deal with this part.